custom/01-be-gone/demo.fth

```diff
@@ -0,0 +1,18 @@
+\ f4711 is a clear indicator that compilation of custom functions was successful
+." f4711( 0, 1, 10, 100, 1000 ) = ( "
+   0 f4711 . ." , "
+   1 f4711 . ." , "
+  10 f4711 . ." , "
+ 100 f4711 . ." , "
+1000 f4711 . ." )"
+CR
+
+\ example of passing passing strings from PForth to custom C code
```

Comment:
Just need one "passing"

Reply:
Thanks for making the other changes.
This is still unresolved. I know that you disagree. But I do not understand why saying "passing passing" is necessary.